docs/flocker-features/pure-storage-configuration.rst

```diff
@@ -0,0 +1,13 @@
+.. _pure-storage-backend:
+
+=================================
+Pure Storage Block Device Backend
+=================================
+
+.. begin-body
+
+Pure Storage provides a Flocker backend plugin.
+
+For more information, including installation, testing and usage instructions, visit the `Pure Storage Flocker Driver on GitHub <https://github.com/sdodsley/purestorage-flocker-driver>`_
```

Comment:
@patrick-east : i think you did test with the latest Flocker bits - the tests that ran on your system are from top of tree, and were not present in 0.3.2.